File: test/data/energyBalance/crop2ml/Algo/py/DiffusionLimitedEvaporation.py
if (deficitOnTopLayers / 1000.0 <= 0):
        
	diffusionLimitedEvaporation = 8.3 * 1000
    
else:
        
	if (deficitOnTopLayers / 1000 < 25):
            
		diffusionLimitedEvaporation = (2 * soilDiffusionConstant * soilDiffusionConstant / (deficitOnTopLayers / 1000.0)) * 1000.0
        
	else:
            
		diffusionLimitedEvaporation = 0

File: test/data/energyBalance/crop2ml/Algo/f90/DiffusionLimitedEvaporation.f90
IF (deficitOnTopLayers / 1000 .LE. 0.0) THEN
	diffusionLimitedEvaporation = 8.3 * 1000
ELSE 
	IF (deficitOnTopLayers / 1000 .LE. 25.0) THEN
		diffusionLimitedEvaporation = (2 * soilDiffusionConstant * soilDiffusionConstant / (deficitOnTopLayers / 1000)) * 1000
	ELSE
		diffusionLimitedEvaporation = 0
ENDIF